contracts/InitialLiquidityPool.sol

```diff
@@ -160,13 +161,39 @@ contract InitialLiquidityPool is Context, IInitialLiquidityPool {
     }
 
     /**
-    * @dev Mints tokens to send to claimants who had bids at IL`        S
+    * @dev Mints tokens to send to claimants who had bids at ILS
     *
     * Emits a {Transfer} event from underlying ERC20 contract   `
     */
-    function claimTokens() override external returns (uint256) {
+    function claimTokens() override external returns (uint) {
         require(_ilsComplete, "ILS has not occurred yet");
 
-        return 0;
+        uint256 existingBid = _bids[_msgSender()];
+
+        require(existingBid > 0,"Token bid unavailable");
+        
+        uint256 tokenAmount = getTokenAmnt(existingBid,_totalBid,_totalSupply);
+
+        //TO DO : - Transfer tokenAmount to msg.sender;
+
+        return tokenAmount;
+        
     }
-}
\ No newline at end of file
+
+    
+    function getTokenAmnt (uint x , uint y,uint z) private pure returns (uint) {
```

Comment:
I think we should keep the names here a bit more context-specific. Like getClaimableTokenAmnt and then naming x, y, and z as the values they represent for readability.